homeassistant-supervised/DEBIAN/postinst

```diff
@@ -65,7 +65,7 @@ info "Restarting docker service"
 systemctl restart "${SERVICE_DOCKER}"
 
 # Check network connection
-while ! ping -c 1 -W 1 ${URL_CHECK_ONLINE}; do
+while ! curl -q ${URL_CHECK_ONLINE} >/dev/null 2>&1 ; do
```

Comment:
`URL_CHECK_ONLINE` has no protocol prefixed. It seems that curl automatically applies a protocol, but ideally we are explicit here:

```suggestion
while ! curl -q "https://${URL_CHECK_ONLINE}" >/dev/null 2>&1 ; do
```

Reply:
Actually, curling `https://checkonline.home-assistant.io/` is slow for me. Ideally we curl `https://checkonline.home-assistant.io/online.txt` directly here. Can you adjust `URL_CHECK_ONLINE` accordingly?

Reply:
> Actually, curling `https://checkonline.home-assistant.io/` is slow for me. Ideally we curl `https://checkonline.home-assistant.io/online.txt` directly here. Can you adjust `URL_CHECK_ONLINE` accordingly?

better to move protocol to URL_CHECK_ONLINE too.

Reply:
Not sure why exactly, but at least here when I curl `https://checkonline.home-assistant.io/`, it is rather slow. Curling `https://checkonline.home-assistant.io/online.txt` is fast. And that is also the location used by the Supervisor online check, so I'd go for the full URL with file.

Reply:
sorry, my fault on replace url :)